pravega/src/main/scala/akka/stream/alpakka/pravega/scaladsl/Pravega.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.pravega.scaladsl
+import akka.stream.scaladsl.{Flow, Keep, Sink, Source}
+import akka.{Done, NotUsed}
+import akka.stream.alpakka.pravega.impl.{PravegaFlow, PravegaSource}
+import akka.stream.alpakka.pravega.ReaderSettings
+import io.pravega.client.ClientConfig
+import io.pravega.client.stream.{EventWriterConfig, Serializer}
+
+import scala.concurrent.Future
+
+object Pravega {
+
+  /**
+   * Messages are read form a Pravega stream.
+   *
+   * Materialized value is a future which completes to Done as soon as Pravega reader in open.
+   */
+  def source[A](scope: String, topic: String)(implicit clientConfig: ClientConfig,
```

Comment:
Do you think it's possible to programmatically create a Pravega `ClientConfig` from typesafe config key/values?  We do something similar in Alpakka Kafka for Consumer and Producer properties.

Reply:
Looks good. I suggest writing a small unit test for this.